tests/puppeteer/campaigns/functional/BO/03_catalog/03_monitoring/05_sortProdutsNoQuantityWithCombination.js

```diff
@@ -0,0 +1,251 @@
+require('module-alias/register');
```

Comment:
The name of the file has an error: `Produts` instead of `Products`.

Reply:
Done thx :)